src/components/LandingPage/Divider/Divider.component.tsx

```diff
@@ -0,0 +1,43 @@
+import loudspeakerIcon from "@/assets/svg/pillars/loudspeaker.svg";
+import relationsIcon from "@/assets/svg/pillars/relations.svg";
+import targetIcon from "@/assets/svg/pillars/target.svg";
+import Image from "next/image";
+
+type Props = {
+  pillar: string;
```

Comment:
```suggestion
  pillar: 'relacje' | 'glos' | 'wplyw';
```

![image](https://github.com/user-attachments/assets/fe07b31d-198b-4ec4-84e2-b9c2261bd401)


W linii 26:
```ts
const { image, text } = ICONS[pillar] || { image: null, text: "" };
```

występuje parametr `pillar`, który jako string może wskazywać na dowolny klucz obiektu `ICONS` (czyli inny niż klucze istniejące: relacje, glos, wplyw). Przypadek podania innego klucza nie jest pokryty, więc build naszej aplikacji się nie udawał. Należy wprowadzić zasugerowaną wyżej zmianę, aby okreslić `pillar` jako Union (zbiór typów) zawierających konkretną listę poprawnych stringów